src/middleware/datasetOverview.middleware.js

```diff
@@ -47,16 +47,16 @@ const fetchEntityCount = fetchOne({
 export const prepareDatasetOverviewTemplateParams = (req, res, next) => {
   const { orgInfo, specification, columnSummary, entityCount, sources, dataset } = req
 
-  const matchingFields = columnSummary[0].matching_field?.split(',') ?? []
-  const nonMatchingFields = columnSummary[0].non_matching_field?.split(',') ?? []
-  const allFields = [...matchingFields, ...nonMatchingFields]
+  const mappingFields = columnSummary[0].mapping_field?.split(';') ?? []
```

Comment:
We need a check (+ associated test case) for `columnSummary.lenght`  (`[][0]` returns `undefined` but `[][0].foo` will throw). We already have some sentry errors like this popping up (I think in the pagination code) from being a bit too optimistic about that.

Reply:
If you want to handle the above in a separate PR, I'm OK with that, just to push this particular fix to prod quickly.